test/integration/targets/apt_key/meta/main.yml

```diff
@@ -1,2 +1,3 @@
 dependencies:
   - prepare_tests
+  - setup_remote_tmp_dir
```

Comment:
This doesn't appear to be used by the `apt_key` test.

Reply:
resolved